src/index.ts

```diff
@@ -56,7 +57,13 @@ async function evaluate(testSuite: EvaluateTestSuite, options: EvaluateOptions =
             };
           } else if (typeof promptInput === 'string') {
             return readPrompts(promptInput);
-          } else {
+          }
+          try {
+            return PromptSchema.parse(promptInput);
+          } catch (error) {
+            console.warn(
+              `Prompt input is not a valid prompt schema: ${error}\nFalling back to serialized JSON as raw prompt.`,
+            );
```

Comment:
@mldangelo Context on this:

When testing via the WebUI (via `npm run dev`, not using .yaml + eval), using the following prompt format for config per prompt:

```yml
prompts:
  - label: 'Prompt #1'
    raw: >-
      a car
    config:
      seed: 123
      image_url: >-
        https://valid.url.to/image.png
```

The prompt object was getting serialized to its string, like:

```js
{
  raw: '{"label":"Prompt #1","raw":"a car","config":{"seed":123,"image_url":"https://valid.url.to/image.png"}}',
  label: '{"label":"Prompt #1","raw":"a car","config":{"seed":123,"image_url":"https://valid.url.to/image.png"}}'
}
```